UKHomeOffice/sgmr-service
# NMSW-34 Handle editing people form when there are autocomplete fields on it

## AC

A bug with the AlphaGov autocomplete causes errors when we prefill the field with a default Value
After discussion with Zabe we agreed to change the UX so the fields from an autocomplete are shown as text with a change link when user is editing

----

## To test

- go to the people page
- add a new person
- start typing in issuing state field
- > country options should be shown
- select a country
- start typing in nationality field
- > country options should be shown
- select a nationality
- refresh the page
- > values should persist
- save person
- click on update for that person
- > issuing state and nationality should be shown as text with a change button for each
- click the change button
- > the autocomplete field should be shown
- start typing in the field
- > the country options should be shown
- select a country
- save the form
- click to update that same person
- > you should see the changed country


----

## Notes
